.github/CONTRIBUTING.md

```diff
@@ -161,3 +162,34 @@ If you're new to open-source or to Briefer, you can start by looking at the issu
 When opening a pull request, please make sure to fill the template with as much information as you can. This will help us understand your contribution and review it more effectively.
 
 After submitting a pull request, please make sure to sign [our CLA (Contributor License Agreement)](./CLA.md) by following the instructions that our bot will provide you. This is a necessary step for us to accept your contribution.
+
+Here’s the fully formatted version of the content with properly labeled code blocks:
+
+<br/>
+
+## :warning: Possible Error Fixes
+Below are some common errors and their possible fixes:
+
+1. **ERR_CERT_AUTHORITY_INVALID**
+this error happens when content is not loaded and in console it displays this error message.
+
+To resolve open a new tab in chrome if chrome is the browser incase other browser check for same options make sure its enabled.
+> chrome://flags/#allow-insecure-localhost
+
+Enable the flag.
+This should resolve the issue.
+
+2. **Jupyter Permissions Issue on Linux/WSL**
+If you're running Jupyter on Linux/WSL and encounter the following error:
+
+> RuntimeError: Permissions assignment failed for secure file:
+'/home/jupyteruser/.local/share/jupyter/runtime/jpserver-8.json'.Got '0o677' instead of '0o0600'.
```

Comment:
```suggestion
```
RuntimeError: Permissions assignment failed for secure file: '/home/jupyteruser/.local/share/jupyter/runtime/jpserver-8.json'. Got '0o677' instead of '0o0600'.
```
```